tests/lib/rules/component-name-in-template-casing.js

```diff
@@ -854,6 +872,39 @@ tester.run('component-name-in-template-casing', rule, {
         </script>
       `,
       errors: ['Component name "TheComponent" is not kebab-case.']
+    },
+    {
+      code: `
+        <template>
+          <router-view />
+        </template>
+      `,
+      options: ['PascalCase', { globals: ['RouterView'] }],
+      output:
+        '\n        <template>\n          <RouterView />\n        </template>\n      ',
+      errors: ['Component name "router-view" is not PascalCase.']
```

Comment:
Please also check line and column of the error.

Reply:
Sorry I don't know what u mean by "line and column of the error" 😥

Reply:
See e.g. https://github.com/vuejs/eslint-plugin-vue/blob/467e85fdf29139050d92e61bd7ea217a53af87cb/tests/lib/rules/no-restricted-component-options.js#L145-L149